devel/ccls/Portfile

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim: fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem                  1.0
+PortGroup                   cmake 1.1
+PortGroup                   github 1.0
+PortGroup                   cxx11 1.1
+PortGroup                   compiler_blacklist_versions 1.0
+
+name                        ccls
+github.setup                MaskRay ccls 0.20190314
+revision                    0
+categories                  devel
+platforms                   darwin
+license                     Apache-2
+
+maintainers                 nomaintainer
+
+description                 C/C++/ObjC language server supporting cross references,\
+                            hierarchies, completion and semantic highlighting
+long_description            ${description}
+
+subport ccls-clang70 {
+    compiler.whitelist          macports-clang-7.0
```

Comment:
So will the newly added clang-8.0 or clang-devel work here? Just whitelisting a single port is also not how we want this handled. If clang >= 7 is required, then you need to blacklist the older versions only. I'm not sure if there is a good why to specific that short of listing all of the older clang ports, which seems non-optimal to me.

Reply:
any ideas how to do it, i'm stumped :( sorry, my first subport.

Reply:
It looks like you need to add this instead:

`compiler.blacklist-append {clang < 700}`

Reply:
Note that this is specifically a `clang 7.0` subport, so `clang 8.0` would presumably have a different subport, and `{clang < 700}` has a completely different meaning.

Reply:
@mojca: So are you saying that the code here is correct?